frontend/src/components/GddSend/TransactionResultLink.vue

```diff
@@ -33,5 +32,20 @@ export default {
       required: true,
     },
   },
+  data() {
+    return {
+      showQrcode: false,
+    }
+  },
+  methods: {
+    showQrCodeButton() {
+      this.showQrcode = !this.showQrcode
+    },
+  },
+  computed: {
+    link() {
+      return `${window.location.origin}/redeem/${this.code}`
```

Comment:
this will be incorrect when Hannes PR is merged

Reply:
is deleted